pom.xml

```diff
@@ -43,6 +43,22 @@
         </developer>
     </developers>
 
+    <dependencies>
+        <!-- Test Dependencies -->
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-core</artifactId>
+            <version>1.3</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-all</artifactId>
```

Comment:
Please do not add dependencies on `mockito-all`. It usually blows up due to the JAR hell. Better to use Mockito-core. Plugin POM 3.30+ also offers the Java 11-compatible version in the dependency management

Reply:
Done. Was able to switch to `mockito-core` without any complications.